Modules/_datetimemodule.c

```diff
@@ -2832,19 +2832,26 @@ date_today(PyObject *cls, PyObject *dummy)
     return result;
 }
 
-/* Return new date from given timestamp (Python timestamp -- a double). */
+/*[clinic input]
+@classmethod
+datetime.date.fromtimestamp
+
+    timestamp: 'O'
+    /
+
+Create a date from a POSIX timestamp.
+
+The timestamp must be a double, e.g. created via time.time() and is
```

Comment:
"must"? Aren't integers accepted too?

Reply:
Yes. I sort of just copied that from the original comment ("Python timestamp -- a double"). That's probably not reasonable anyway since this is the docstring for the Python API and thus should not use a C type. Is "The timestamp is a number, e.g. ..." ok?

Reply:
> Is "The timestamp is a number, e.g. ..." ok?

LGTM.